pdc_client/__init__.py

```diff
@@ -125,7 +125,10 @@ class to perform requests.
             if insecure is not None:
                 sys.stderr.write("Warning: '%s' option is deprecated; please use '%s' instead\n" % (
                     CONFIG_INSECURE_KEY_NAME, CONFIG_SSL_VERIFY_KEY_NAME))
-                ssl_verify = not insecure
+                if insecure:
+                    ssl_verify = not insecure
+                else:
+                    ssl_verify = ssl_verify or not insecure
```

Comment:
@ycheng-aa ; I undstand your means~! In my opinion, in current status;I consider that the `insecure` is more high priority than `ssl_verify`, do you think so? I have update the patch, which maybe more simple and clear. Please switch to new patch and help  to review it. 
I will send you and @simozhan a email to discuss about that I want to remove "insecure" key and only "ssl_verify" key in client.